docs/blog/introducing-rocket.md

```diff
@@ -3,6 +3,7 @@ title: Introducing rocket - effective static content with some javascript
 published: false
 description: Write Interactive Demos Using Markdown and JavaScript
 date: 2020-07-30
+layout: blog-details.njk
```

Comment:
hrm... not sure why this is showing up here? There are a bunch of changes I didn't do? 🙈 I just forked from dakmor/rocket#master branch 😕